src/main/java/org/jabref/gui/importer/ImportEntriesViewModel.java

```diff
@@ -78,10 +82,13 @@ public ImportEntriesViewModel(BackgroundTask<ParserResult> task,
         this.message.bind(task.messageProperty());
 
         task.onSuccess(parserResult -> {
+
             // store the complete parser result (to import groups, ... later on)
             this.parserResult = parserResult;
+
+            List<BibEntry> resultEntries = parserResult.getDatabase().getEntries().stream().map(cleanup::doPostCleanup).collect(Collectors.toList());
```

Comment:
An idea would be to move this cleanup to the `ImportHandler` for consistency reasons.

Reply:
Is already part of the importHandler for importing external files
https://github.com/JabRef/jabref/blob/bb011c9313367a28990ae213b3920fe6cd10d1dc/src/main/java/org/jabref/gui/externalfiles/ImportHandler.java#L165-L169

Reply:
Okay, then I don't get why you added it in the dialog. The `buildImportHandlerThenImportEntries` method uses the import handler to import entries.

Reply:
Ah thanks, then it can be removed here. Totally overlooked that this is called.